app/templates/explore.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="ui stackable grid">
+  <div class="four wide column">
+    {{explore-side-bar}}
+  </div>
+  <div class="twelve wide column">
+    {{event-list events=model}}
+  </div>
+</div>
```

Comment:
Add `{{modals/event-share-modal isOpen=isShareModalOpen event=eventToShare}}` after this for event sharing to work.